File: Programmers/JAVA/lv1/lv1_서울에서 김서방 찾기.java
https://programmers.co.kr/learn/courses/30/lessons/12919

class Solution {
    public String solution(String[] seoul) {
        String answer = "";
        
        for(int i = 0; i < seoul.length; i++) {
            if(seoul[i].equals("Kim")) {
                answer = "±è¼­¹æÀº " + i + "¿¡ ÀÖ´Ù";
                break;
            }
        }
                
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_서울에서 김서방 찾기.cs
https://programmers.co.kr/learn/courses/30/lessons/12919?language=csharp

using System;

public class Solution {
    public string solution(string[] seoul) {
        return String.Format("김서방은 {0}에 있다", Array.FindIndex(seoul, i => i == "Kim"));
    }
}